pandas/tseries/offsets.py

```diff
@@ -1742,21 +1747,22 @@ class LastWeekOfMonth(DateOffset):
 
     Parameters
     ----------
-    n : int
-    weekday : {0, 1, ..., 6}
+    n : int, default 1
+    weekday : {0, 1, ..., 6}, default None
         0: Mondays
         1: Tuesdays
         2: Wednesdays
         3: Thursdays
         4: Fridays
         5: Saturdays
         6: Sundays
+
     """
 
-    def __init__(self, n=1, normalize=False, **kwds):
+    def __init__(self, n=1, normalize=False, weekday=None):
```

Comment:
is there a test for this as weekday=None? isn't that an error?

Reply:
It is an error, though a different one in Py2 vs Py3.  In the status quo not passing a weekday explicitly will raise a KeyError.  Default to Monday I guess?

Reply:
no if weekday is None, just raise a value error, in reality this should actually be the first (and a required parameter), but I guess that would be hard to change. Don't set defaults, raise

Reply:
Are you suggesting it should be `__init__(self, weekday, n=1, normalize=False)`?  I see some appeal to that, but there is a lot of symmetry with other signatures all starting with `(self, n=1, normalize=False, ...)`.

Reply:
see my comment above. this is ok for now. actually weekday is optional, so this is fine.

Reply:
Two things.  First, here and in other places you say "this is fine" but I don't know whether you're referring to the PR or the thing the PR is fixing.  Based on the go-ahead you've given below to look over timeseries.rst I'm assuming you mean the PR version is fine, just heads up for ways I'm liable to get confused next time around.

> actually weekday is optional,

Under the status quo, failing to pass `weekday` raises a `KeyError`.  In the PR, not passing anything will raise a ValueError in py2 and TypeError in py3.  (The same errors would result in the status quo if a user passed `None` explicitly)

Reply:
I never mean this entire PR is fine. If that was the case I wouldn't have any other comments. Individual comments are revering to the specific line/section where I make it.